resources/site-packages/quasar/daemon.py

```diff
@@ -181,6 +181,15 @@ def start_quasard(**kwargs):
         kwargs["env"] = env
         kwargs["close_fds"] = True
 
+    wait_counter = 1
+    while xbmc.getCondVisibility('Window.IsVisible(10140)'):
+        if wait_counter > 256:
```

Comment:
> We're not programs Gerty, we're people

Go with 300, as in 5 minutes, just increase the counter by 1, and `time.sleep(1)`
256 seconds means nothing to humans...